Incorrect label name used in pr-backport workflow

<!--
⚠️ Public content warning ⚠️
Issues created in this repository will be publicly available on the Internet. Please do not include any sensitive or internal information in your issue.

🔒 Reporting security vulnerabilities 🔒
Do not create issues about security vulnerabilities. Please consult our security policy for instructions about how to safely report a security issue: https://github.com/az-digital/az_quickstart/security/policy
-->

## Problem/Motivation
Our GH actions workflow that automatically creates backport PRs for the 2.x branch adds labels to the PRs it generates but one of the labels referenced in the workflow no longer exists (`2.x-only`).

### Describe the bug
```
Run # Prepare PR body
could not add label: '2.x-only' not found
Error: Process completed with exit code 1.
```

## Proposed resolution
Update the label reference in the workflow.

--- a/.github/workflows/pr-backport.yml
+++ b/.github/workflows/pr-backport.yml
@@ -72,5 +72,5 @@ jobs:
             --head "backport/pr-${{ github.event.pull_request.number }}-2.x" \
             --title "${{ github.event.pull_request.title }} (2.x backport of #${{ github.event.pull_request.number }})" \
             --body "$BODY" \
-            --label "2.x-only,backport" \
+            --label "2.x only,backport" \
             $([ "${{ steps.backport.outputs.status }}" == "conflicts" ] && echo "--draft" || echo "")

